internal/jobservice/eventstojobs/eventstojobs.go

```diff
@@ -87,7 +87,11 @@ func (eventToJobService *EventsToJobService) streamCommon(ctx context.Context, t
 			select {
 			case <-ctx.Done():
 				log.Errorf("context is done on %s/%s", eventToJobService.queue, eventToJobService.jobSetId)
-				return nil
+				_, err := eventToJobService.jobServiceRepository.UnsubscribeJobSet(
```

Comment:
I think this is not what we want actually.  I went back and forth but we still want the list of jobsets

Reply:
Otherwise once we unsubscribe we no longer have the list and what we really want is to reconnect to the existing jobset.

Reply:
Then I don't know how to solve this, because with the current logic we are never going to clear a subscription.

Reply:
I think when we unsubscribe we close the event connection.  Isn't that enough?